src/main/java/persistence/meta/MetaDataColumns.java

```diff
@@ -30,11 +32,17 @@ public static MetaDataColumns of(Class<?> clazz, Dialect dialect) {
         .filter(column -> !column.isNotPrimaryKey())
         .findFirst().orElseThrow(() -> new RuntimeException("ID 필드가 없습니다."));
 
-    return new MetaDataColumns(metaColumns, primaryKeyColumn);
+    Relation relation = metaColumns.stream()
+        .filter(column -> column.hasRelation())
+        .map(column -> column.getColumnRelation())
+        .findAny().orElse(new MetaDataColumnEmptyRelation());
```

Comment:
```suggestion
        .filter(MetaDataColumn::hasRelation)
        .map(MetaDataColumn::getColumnRelation)
        .findAny().orElseGet(MetaDataColumnEmptyRelation::new);
```
이런식으로 사용해 보는건 어떠신가요?
orElse와 orElseGet 의 차이도 알아 보셨으면 좋겠어요🙂

Reply:
이런... 마구쓰다보니깐 의도치않게 get을 사용했네요 

한번더 마음에 새기고 내용 확인했습니다!


method reference로 구현하고 내용도 복습했습니다. 
확인해주셔서 감사합니다!